searx/sxng_locales.py

```diff
@@ -56,6 +56,7 @@
     ('fr-CA', 'Français', 'Canada', 'French', '\U0001f1e8\U0001f1e6'),
     ('fr-CH', 'Français', 'Suisse', 'French', '\U0001f1e8\U0001f1ed'),
     ('fr-FR', 'Français', 'France', 'French', '\U0001f1eb\U0001f1f7'),
+    ('gl', 'Galego', '', 'Galician', '\U0001f310'),
```

Comment:
can you remove this hunk from the patch / I assume its a leftover when you updated the `searx/data/wikidata_units.json`

Reply:
Oh yeah sorry, I didn't notice `make data.all` also modifies `sxng_locales.py`, I've only reverted everything in `searxng/data`.